Analysis/Tasks/PWGHF/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ o2_add_dpl_workflow(qa-simple
 
 o2_add_dpl_workflow(hf-track-index-skims-creator
                     SOURCES HFTrackIndexSkimsCreator.cxx
-                    PUBLIC_LINK_LIBRARIES O2::Framework O2::AnalysisDataModel O2::AnalysisCore O2::DetectorsVertexing ROOT::EG
+                    PUBLIC_LINK_LIBRARIES O2::Framework O2::AnalysisDataModel O2::AnalysisCore O2::DetectorsVertexing O2::AnalysisTasksUtils ROOT::EG 
```

Comment:
Trailing spaces should be removed (ideally by the editor automatically).

Reply:
done